packages/blade/src/components/VisuallyHidden/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './VisuallyHidden';
+export * from './VisuallyHidden.d';
+export { default } from './VisuallyHidden';
```

Comment:
can you also add it to exports on `src/components/index.ts`?

Reply:
Right